apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,281 @@
+        messageInterpolation: { limit: READINGS_LIMIT, sentCount },
+    }),
+    ORGANIZATION_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: ORGANIZATION_NOT_FOUND,
+        message: 'Organization not found',
+        messageForUser: 'api.meter.registerMetersReadings.ORGANIZATION_NOT_FOUND',
+    },
+    PROPERTY_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: PROPERTY_NOT_FOUND,
+        message: 'Property not found',
+        messageForUser: 'api.meter.registerMetersReadings.PROPERTY_NOT_FOUND',
+    },
+    INVALID_METER_VALUE: (key, value) => ({
+        code: BAD_USER_INPUT,
+        type: INVALID_METER_VALUE,
+        message: 'Invalid meter value',
+        messageForUser: 'api.meter.registerMetersReadings.INVALID_METER_VALUE',
+        messageInterpolation: { key, value },
+    }),
+    MULTIPLE_METERS_FOUND: (count) => ({
+        code: BAD_USER_INPUT,
+        type: MULTIPLE_METERS_FOUND,
+        message: 'Multiple meters found',
+        messageForUser: 'api.meter.registerMetersReadings.MULTIPLE_METERS_FOUND',
+        messageInterpolation: { count },
+    }),
+}
+
+function toISO (str) {
+    if (isEmpty(str)) {
+        return undefined
+    }
+
+    return dayjs(str).toISOString()
+}
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
+    if (!(isString(value) || isNumber(value) || isUndefined(value))) return null
+    if (isUndefined(value)) return value
+
+    const transformedValue = String(value).trim().replaceAll(',', '.')
+    if (isEmpty(transformedValue)) return undefined
+    return String(Number(transformedValue))
+}
+
+/**
+ * @param {string | null | undefined} value
+ * @return {boolean}
+ */
+function validateMeterValue (value) {
+    if (!isString(value) && !isUndefined(value)) return false
+    if (isUndefined(value)) return true
+
+    return !isEmpty(value) && !isNaN(Number(value)) && isFinite(Number(value)) && Number(value) >= 0
+}
+
+const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReadingsService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsMeterMetaInput {' +
+                'numberOfTariffs: Int,' +
+                'place: String,' +
+                'verificationDate: String,' +
+                'nextVerificationDate: String,' +
+                'installationDate: String,' +
+                'commissioningDate: String,' +
+                'sealingDate: String,' +
+                'controlReadingsDate: String' +
+                '}',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsReadingAddressMetaInput { unitType: String!, unitName: String!, globalId: String }',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsReadingInput {' +
+                'address: String!,' +
+                'addressMeta: RegisterMetersReadingsReadingAddressMetaInput!,' +
```

Comment:
We should be more consistent with naming?

Reply:
renamed